### PR TITLE
Gameplay game end implementation

### PR DESCRIPTION
**Pull request by skystar-p**

### CODE REVIEW

File: backend/websocket/consumers/gameplay_consumers.py [hunk 1]
```diff
@@ -1002,6 +1070,12 @@ def gameplay_continue_consumer(message):
         if player['continue'] is True:
             continue_count += 1
 
+    reply_channel.send(response(
+        {},
+        nonce=nonce,
+    ))
+    cache.set('room:' + room_id, room)
```
Comment (tirr-c) [inline backend/websocket/consumers/gameplay_consumers.py]: `room`이 변하지 않는데 저장할 필요 없을 듯

Comment (tirr-c) [reply backend/websocket/consumers/gameplay_consumers.py]: 참조를 하고 있었다...

**Comment by tirr-c** [comment]
@web-mighty-homu r+

**Comment by web-mighty-homu** [comment]
:pushpin: Commit d677029 has been approved by `tirr-c`

**Comment by web-mighty-homu** [comment]
:hourglass: Testing commit d67702994746233910238b5cb7a140b411d16cef with merge da8e7465d571efd6a808af8ab8b2c0a29a840028...

**Comment by web-mighty-homu** [comment]
:sunny: Test successful - [status-travis](https://travis-ci.org/swsnu/swpp17-team2/builds/317139825?utm_source=github_status&utm_medium=notification)
Approved by: tirr-c
Pushing da8e7465d571efd6a808af8ab8b2c0a29a840028 to master...

**Pull request merged**